manifests/charts/istio-control/istio-discovery/templates/clusterrole.yaml

```diff
@@ -116,6 +116,14 @@ rules:
     resources: ["secrets"]
     verbs: ["get", "watch", "list"]
 
+  # Maistra specific
+  - apiGroups: ["maistra.io"]
+    resources: ["servicemeshmemberrolls"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: ["route.openshift.io"]
+    resources: ["routes", "routes/custom-host"]
+    verbs: ["get", "list", "watch", "create", "delete", "update"]
```

Comment:
Actually, we were changing roles only in the istio-operator in the past, but when I worked on https://github.com/maistra/istio/pull/458, I had to modify this file. Then we discussed how to handle changes in helm manifests and there was an idea to remove [patch-charts.sh](https://github.com/maistra/istio-operator/blob/maistra-2.4/build/patch-charts.sh), but we never moved those changes from istio-operator to istio.
I also considered removing this change, because now we apply custom roles from [tests/integration/servicemesh/maistra/testdata/roles.yaml](https://github.com/maistra/istio/blob/maistra-2.4/tests/integration/servicemesh/maistra/testdata/role.yaml), so we no longer need these roles here. If we will decided to do that, we will have to revert this [commit](https://github.com/maistra/istio-operator/pull/1042) as well.
@luksa could you also share your opinion?

Reply:
I thought about it more and I would prefer to keep it here, even though I don't like it, because it's already in maistra/istio 2.2 - 2.4, so maybe it does not make sense to change it once again...